External/FEXCore/Source/Interface/Core/SignalDelegator.cpp

```diff
@@ -542,24 +568,13 @@ namespace FEXCore {
     // Additionally we must always listen to SIGNAL_FOR_PAUSE
     // This technically forces us in to a race but should be fine
     // SIGBUS and SIGILL can't happen so we don't need to listen for them
-    sigaddset(&HostSet, SIGNAL_FOR_PAUSE);
+    //sigaddset(&HostSet, SIGNAL_FOR_PAUSE);
```

Comment:
Is this commented out on purpose?

Reply:
Yea, I don't trust it and since it is a debug only feature then I'm commenting it out for now.